package.json

```diff
@@ -178,6 +178,7 @@
 	},
 	"devDependencies": {
 		"@types/chai": "^4.2.11",
+		"@types/download": "^6.2.4",
```

Comment:
sure that it is a devDependencies? No tests were added so i'm surprised.

Reply:
Definitely devDependencies since it's used in src/extensionFunctions.ts and it won't build without errors if we remove it. :)

Reply:
if it is used in src I think it should be placed in "dependencies" section, not "devDependencies"